src/projects/automation/workflows/exclusive.md

```diff
@@ -0,0 +1,42 @@
+# Exclusive Node
+
+An exclusive node allows you to put a decision point in your workflow. You can place conditions on each path branching out from the exclusive node and the it will only execute paths where the condition is `true`.
+
+The conditions can use workflow inputs, activity outputs and datastore variables in them.
+
+They can also be used to introduce loops into your workflow as well as error handling.
+
+There are several key operators you can use to create your conditions:
+
+| Operator | Description |
+| --- | --- |
+| `==` | If right hand side equals left hand side then it is true |
+| `!=` | If right hand side does not equal left hand side then it is true |
+| `<` | If right hand side is greater than left hand side then it is true |
```

Comment:
While this (and all of these) are technically true, is there a reason we're inverting the _natural_ order? For example this operator is referred to as `less than`, but the wording you have used is `greater than`.